clientv3/README.md

```diff
@@ -72,6 +72,14 @@ if err != nil {
 }
 ```
 
+## Metrics
+
+etcd client provides metrics reporting. This can enable the operators to know how clients are performing and which client generates the most load.
```

Comment:
How about?

etcd uses [go-grpc-prometheus](https://github.com/grpc-ecosystem/go-grpc-prometheus) to expose various client RPC metrics. Separate Prometheus handler needs to be started in client side. Examples can be found [here](https://github.com/coreos/etcd/blob/master/clientv3/example_metrics_test.go).

/cc @heyitsanthony